### DISCUSSION TITLE
Win 11 - new updates with the new ExploerPatcher version

### DISCUSSION THREAD

**Discussion by romafrogs** [valinet/ExplorerPatcher#2013] [Q&A]
If im installing the new (2 weeks ago) patcher version

Can i install ALL the new win 11 updates? including preview?

**Comment by TroelsWibergJensen** [comment]
No i think not. For me explorerpatcher crashed after installing build 22621.2215

**Comment by romafrogs** [comment]
So I should just stop win 11 updates for good? :( or only the preview ones?

**Reply by pyrates999** [reply]
Uninstall preview updates if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

Preview updates are not supported by EP.

**Reply by Poopooracoocoo** [reply]
It's your choice. You could go to OS Build 22621.1992. You could stay on the current GA release, 22621.2314, and have a broken EP experience. You may even want to go back to Windows 10.

**Comment by ghost** [comment]
You don't need ExplorerPatcher anymore.

**Comment by romafrogs** [comment]
> You don't need ExplorerPatcher anymore.

Why not?

**Comment by Poopooracoocoo** [comment]
No. As per the wiki, EP explicitly does not support Insider builds and preview updates. And currently it does not work properly on the latest GA release of Windows 11, OS Build 22621.2134. If you're already on this build, see if you can uninstall KB5029263. The latest version of EP only claims support up to OS Build 22621.1992, as per the version number and description of the release on GitHub

**Reply by romafrogs** [reply]
I have KB5029263 insralled with the EP build. and it works great
after i did :
Uninstall EP
Remove these registry keys (you can save the below as a .reg file if you prefer)...
Windows Registry Editor Version 5.00

[-HKEY_CURRENT_USER\Software\ExplorerPatcher]
[-HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher]
Reboot
Reinstall EP (the latest / same version you that was installed previously will work)

**Reply by pyrates999** [reply]
All you've done is removed all of EP's settings stored in the registry, thus making EP not change anything in windows.  You might as well not have EP even installed.

**Reply by Twisted-Code** [reply]
Out of curiosity, have y'all considered having EP do a version check before trying to load/inject itself/whatever? It's been [known to make systems completely unusable after updates](https://answers.microsoft.com/en-us/windows/forum/all/kb5029263-caused-systems-to-become-unusable-due-to/ef6462d9-8902-4caf-8f17-21ef7a0f8642)[\*], and not everyone is tech savvy enough to double check for compatibility before updating Windows (especially when Windows likes to update itself _**without asking**_; like today I would've checked, if I'd been given the chance, but my update pause ran out). To be frank, I mostly have EP installed so that 7 taskbar tweaker can run, so risking such a breakage (again) for something I might be able to run with another hack seems foolish.

For now, the Taskview button seems to be broken for me, so until that is working again, I think I'll just deal with the Windows 11 experience for a bit. It's bad, but not bad enough to deal with all the pain that can come from rolling back Windows Updates. (Perhaps this is the argument I've needed all along to get myself to switch over to Linux. I'm going to look into that…)

NB: I can confirm the symptoms described in the link, though I did not realize the correlation at the time and, among other attempted remedies, reset my system to its factory image to solve it… I have much frustration at the realization I could've avoided resetting if I'd simply been able to stop EP from trying to load. It seems like it was even loading in safe mode for some reason! Why?!

**Comment by mistertheloaf** [comment]
> The latest version of EP only claims support up to OS Build 22621.1992

Yes.  ExplorerPatcher is an excellent program, and it works very well, **BUT**, patching explorer.exe is **VERY** tricky business, so EP only works properly with the version designated in the EP version number -- in this case 22621.1992.

Trying to use EP on anything newer than 22621.1992 may cause crashes and other problems.

**Comment by romafrogs** [comment]
Is there a way to just click somewhere so that specific updates i choose wont be installed? forever?
so i could keep updating other updates?

**Reply by tallman69** [reply]
If you are talking about Windows 11 updates, the answer is no, the best you can do is pause updates for up to a month.  If you don't want Windows 11 updated the only thing you can do is disconnect your computer from the internet.

**Comment by newtekie1** [comment]
Unfortunately EP only supports a version of Win11 that is over a year old and it doesn't seem like there is any interest in getting it to support anything newer. It's been broken with the latest non-preview releases for over a month now and it is very obvious no work is being done to  fix it. At this point we might as well consider EP as abandoned.

**Reply by tallman69** [reply]
I would not be so quick to say that EP has been abandoned.  It has only been a couple of weeks since the last EP update came out.  It is unfortunate that a number of users are having issues with EP and the lates build of Windows 11.  I feel fortunate that I am not having any issues with my three computers that are running EP on 2261.2134.  It is a risk we take when using an application like EP that things will go awry when trying to alter a very complicated and ever-changing OS like Windows 11.  But it is risk I am willing to live with at this point.

**Reply by mistertheloaf** [reply]
>> Unfortunately EP only supports a version of Win11 that is over a year old

That is not true.  The latest EP supports Win11 22621.1992 which was released 2 months ago.
